content/running_spark_apps_with_emr_on_spot_instances/scaling_emr_cluster.md

```diff
@@ -0,0 +1,56 @@
+---
+title: "Scaling EMR cluster"
+weight: 95
+---
+
+While you can always manually adjust the number of core or task nodes (EC2 instances) in your Amazon EMR cluster, you can also use the power of EMR auto-scaling to automatically adjust the cluster size in response to changing workloads without any manual intervention.
+
+In this section, we are going to enable automatic scaling for the cluster using **[Amazon EMR Managed Scaling](https://aws.amazon.com/blogs/big-data/introducing-amazon-emr-managed-scaling-automatically-resize-clusters-to-lower-cost/)**. With EMR Managed scaling you specify the minimum and maximum compute limits for your cluster and Amazon EMR automatically resizes EMR clusters for best performance and resource utilization. EMR Managed Scaling constantly monitors key metrics based on workload and optimizes the cluster size for best resource utilization. 
+
+{{% notice note %}}
+EMR Managed Scaling is supported for Apache Spark, Apache Hive and YARN-based workloads on Amazon EMR versions 5.30.1 and above.
+{{% /notice %}}
+
+### Enable Managed Scaling
+
+1. In your EMR cluster page, in the AWS Management Console, go to the **Summary** tab.
+1. Copy the **ID** from the **Summary** tab.
+1. Open the shell terminal in your Cloud9 enivronment that you created at the beginning of this workshop.
+1. Run the command after replacing the **CLUSTER-ID** with the one you copied earlier.
+
+```Bash
+aws emr put-managed-scaling-policy \
+    --cluster-id CLUSTER-ID  \
+    --managed-scaling-policy "ComputeLimits={
+            UnitType=InstanceFleetUnits,
```

Comment:
This is incorrect, you have to set the managed scaling as per Instance fleets numbers. 
MaximumCoreCapacityUnits = 4 (we don't want to scale core)
MaximumOnDemandCapacityUnits = 4 (core runs on OD)
MinimumCapacityUnits = Core + Tasks = (4 + 32) = 36
MaximumCapacityUnits = Core + Tasks * 2 = (4+ 32*2) = 68


![image](https://user-images.githubusercontent.com/70941399/173979113-8d5c7621-4bb0-48d6-b226-1abda099b4d1.png)

Reply:
Thanks @jagpk ! Instructions are to choose **4 Spot units** for Core Nodes. I'll keep the On-Demand value to 0 for now unless we want to change Core Nodes to use On-Demand instances.